gulpfile.js

```diff
@@ -137,8 +139,19 @@ gulp.task("watch", function () {
   gulp.watch(dirs.img + "/**/*.*", ["images"]);
 });
 
+gulp.task("make-war", function () {
```

Comment:
Can we include the version string from package.json here? Or is this supposed to be included on another place?